ceph-releases/ALL/rhel7/daemon/container.yaml

```diff
@@ -0,0 +1,17 @@
+compose:
+  packages:
```

Comment:
Keeping this list up-to-date by hand is going to be painful. I'd rather we omitted the `packages` list entirely here and composed the entire contents of the Brew tag. It should not take that much longer, and it's computer time vs the human cost of keeping this updated.

If we had something that could parse our Dockerfile and determine this list automatically from "yum install" statements, that would be great, but that is not easy.

Reply:
Without compose:packages but with pulp_repos:true, only content sets (IOW released content) will be pulled in.  This would mean that the RPMs -- even the Ceph ones -- would have to be *released* before you could even *build* a container with them.  Therefore, using ODCS is not practical without compose:packages.

The other alternative to a repo file is to rely solely on the build tag inheritance, which would need to be adjusted even more.